clients/spark/build.sbt

```diff
@@ -126,7 +138,7 @@ lazy val s3UploadSettings = Seq(
     (assemblyOutputPath in assembly).value ->
       s"${name.value}/${version.value}/${(assemblyJarName in assembly).value}"
   ),
-  s3Upload / s3Host := "treeverse-clients-us-east.s3.amazonaws.com",
+  s3Upload / s3Host := "bobotreeverse-clients-us-east.s3.amazonaws.com",
```

Comment:
What is bobotreeverse? where is this bucket created?

Reply:
Ouch. That never should have gone out of my computer, of course.